cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/LaunchStackHandler.java

```diff
@@ -53,6 +53,7 @@ public void accept(Event<LaunchStackRequest> launchStackRequestEvent) {
         try {
             CloudConnector connector = cloudPlatformConnectors.get(cloudContext.getPlatformVariant());
             AuthenticatedContext ac = connector.authentication().authenticate(cloudContext, request.getCloudCredential());
+            connector.credentials().create(ac);
```

Comment:
Is it not necessary to check the returned CloudCredentialStatus? It can be failed.